test/e2e/framework/pv_util.go

```diff
@@ -613,21 +620,29 @@ func MakePersistentVolume(pvConfig PersistentVolumeConfig) *v1.PersistentVolume
 	}
 }
 
-// MakePersistentVolumeClaim returns a PVC definition based on the namespace.
-// Note: if this PVC is intended to be pre-bound to a PV, whose name is not
-//   known until the PV is instantiated, then the func CreatePVPVC will add
-//   pvc.Spec.VolumeName to this claim.
+// MakePersistentVolumeClaim returns a PVC API Object based on the PersistentVolumeClaimConfig.
 func MakePersistentVolumeClaim(cfg PersistentVolumeClaimConfig, ns string) *v1.PersistentVolumeClaim {
-	// Specs are expected to match this test's PersistentVolume
 
 	if len(cfg.AccessModes) == 0 {
-		e2elog.Logf("AccessModes unspecified, default: ReadWriteOnce (RWO).")
 		cfg.AccessModes = append(cfg.AccessModes, v1.ReadWriteOnce)
 	}
 
+	if len(cfg.ClaimSize) == 0 {
+		cfg.ClaimSize = "2Gi"
```

Comment:
is this documented anywhere? seems a bit magical to me

Reply:
Documented here: https://github.com/kubernetes/kubernetes/pull/80382/files#diff-420013911dac4a3b0f16035b6e90a571R96

Less magical than the previous defaulting even :)
There was logging everytime we defaulted that I removed: https://github.com/kubernetes/kubernetes/pull/80382#discussion_r305955175

Reply:
:+1: